### DISCUSSION TITLE
Does Turborepo work with Renovate?

### DISCUSSION THREAD

**Discussion by richardkazuomiller** [vercel/turborepo#1082] [Help] [answered]
I want to switch from Lerna to something else, but I need to continue using [Renovate](https://www.whitesourcesoftware.com/free-developer-tools/renovate/) to update dependencies. Does anyone have experience making them work together?

**Comment by lodmfjord** [comment, marked as answer]
It works. Doesn't really change anything to my knowledge except maybe making the CI faster when testing the project with new deps.

**Comment by maksnester** [comment]
> It works

I see that only root package.json gets picked up by renovate, any special configs to make it go through the npm workspaces?
